src/components/SponsorPaymentModal.vue

```diff
@@ -48,47 +66,52 @@
 </template>
 
 <script>
-import { mapState, mapActions } from 'vuex';
-import SimpleWallet from '@/libs/SimpleWallet';
-import API from '@/util/api';
-import QrCode from '@xkeshi/vue-qrcode';
+import { mapState, mapActions } from "vuex";
+import SimpleWallet from "@/libs/SimpleWallet";
+import API from "@/util/api";
+import QrCode from "@xkeshi/vue-qrcode";
 
-const walletHelper = new SimpleWallet('Crypto Meetups');
+const walletHelper = new SimpleWallet("Crypto Meetups");
 
 export default {
-  name: 'SponsorPaymentModal',
-  props: ['countryName', 'transaction'],
+  name: "SponsorPaymentModal",
+  props: ["countryName", "transaction"],
   components: {
-    QrCode,
+    QrCode
   },
   data: () => ({
-    isScatterPaying: false,
+    isScatterPaying: false
   }),
   computed: {
-    ...mapState(['isScatterConnected', 'scatterAccount', 'isScatterLoggingIn']),
+    ...mapState(["isScatterConnected", "scatterAccount", "isScatterLoggingIn"]),
+    isPhone() {
+      //copied from stackoverflow
+      return /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
+    },
     walletTransferData() {
       const payload = {
         to: this.transaction.to,
         amount: this.transaction.amount,
-        contract: 'eosio.token',
-        symbol: 'EOS',
+        contract: "eosio.token",
+        symbol: "EOS",
         precision: 4,
         dappData: this.transaction.memo,
-        desc: 'Crypto Meetup - Become Country Sponsor',
-        expired: Math.floor(Date.now() / 1000 + 10 * 60),
+        desc: "Crypto Meetup - Become Country Sponsor",
+        expired: Math.floor(Date.now() / 1000 + 10 * 60)
       };
       return JSON.stringify(walletHelper.transfer(payload));
-    },
+    }
   },
   methods: {
-    ...mapActions(['loginScatterAsync', 'updateLandInfoAsync']),
+    ...mapActions(["loginScatterAsync", "updateLandInfoAsync"]),
     paidWithWalletApp() {
       this.updateLandInfoAsync();
       this.$dialog.alert({
-        type: 'is-black',
-        title: '恭喜你成功转账',
-        message: '30秒内自动刷新数据，即可确认你是否为新地主。一切以区块链上交易记录为准，购买地皮失败则退款。',
-        confirmText: '好的',
+        type: "is-black",
```

Comment:
Prefer to use toast instead

Reply:
I don't think that my code 🙄 
👇👇👇👇👇👇
[history](https://github.com/crypto-meetup-dev/cryptomeetup-portal/commit/43d7e75e867973e01d2508110151bbf20e356d66#r31134939)

You started this `SponsorPaymentModal.vue` file with this `alert`, buddy. Can you do this yourself? Thank you

Reply:
It's not my code as well, I just do renames :(